src/ol/map.js

```diff
@@ -585,27 +570,25 @@ ol.Map.prototype.disposeInternal = function() {
  *     the {@link ol.layer.Layer layer} of the feature and will be null for
  *     unmanaged layers. To stop detection, callback functions can return a
  *     truthy value.
- * @param {S=} opt_this
+ * @param {S=} opt_this Value to use as this when executing callback.
+ * @param {olx.ForEachFeatureOptions=} opt_options Optional options.
```

Comment:
The compilers demands all optional parameters to appear at the end of the signature. To be concise with the other uses of opt_this i left it right behind the callback and inserted the opt_options behind that.